datacats/docker.py

```diff
@@ -103,14 +103,15 @@ def _get_docker():
 
 class WebCommandError(Exception):
 
-    def __init__(self, command, container_id, logs):
+    def __init__(self, command, container_id, logs, ssh=False):
```

Comment:
so you are suggesting using this ssh argument as a flag to differentiate between the remotecommand-specific problems and all others?